load/loader_test.go

```diff
@@ -306,8 +307,11 @@ func TestCreateChannelsAndPartitions(t *testing.T) {
 }
 
 func TestWork(t *testing.T) {
+
 	br := &CommonBenchmarkRunner{}
 	b := &testBenchmark{}
+	br.ReportingMetricsPort = 0
```

Comment:
Why this is needed?

Reply:
I was comparing the port to check if it exists and then I need to set it to zero. Not sure what is the best place to set this default but I'll try to move it to the initializer and maybe it will not be needed if we just use the "full host" as a config.